io/zenoh-transport/tests/unicast_transport.rs

```diff
@@ -669,6 +850,112 @@ fn transport_unicast_ws_only() {
     task::block_on(run(&endpoints, &endpoints, &channel, &MSG_SIZE_ALL));
 }
 
+#[cfg(all(feature = "transport_ws", feature = "shared-memory",))]
+#[test]
```

Comment:
Instead of duplicating every test, wouldn't be easier to execute `run_with_net` and `run_with_shm` in already existing tests? In this way you also do not need to care about port number collisions.